repondeur/zam_repondeur/templates/amendements.html

```diff
@@ -1,152 +1,250 @@
+        <div>
+            <h3>Importer une liasse XML</h3>
+            <p>Importer des amendements à partir d'un fichier XML provenant
+            de l’Assemblée nationale.</p>
+            <form action="{{ request.resource_url(context.parent, 'import_liasse_xml') }}" enctype="multipart/form-data" method="post" id="import-liasse-xml" class="align-items-center">
+                <div class="custom-file">
+                    <input type="file" class="custom-file-input" id="liasse" name="liasse">
+                    <label class="custom-file-label" for="liasse">Fichier XML</label>
+                </div>
+                <div role="group">
+                    <input type="submit" name="upload" class="button" value="Charger la liasse">
+                </div>
+            </form>
         </div>
-    </div>
+        <div>
+            <h3>Télécharger ce tableau</h3>
+            <p>Télécharger le tableau affiché ci-dessous de tous les amendements déposés ainsi que de leurs réponses, par ordre de passage.</p>
+            <div role="group" aria-label="Format de téléchargement">
+                <a class="button" href="{{ request.resource_url(context.parent, 'download_amendements', query={'format': 'csv'}) }}">
+                    <i class="fa fa-table"></i>
+                    Format CSV
+                </a>
+                <a class="button" href="{{ request.resource_url(context.parent, 'download_amendements', query={'format': 'xlsx'}) }}">
+                    <i class="fa fa-file-excel"></i>
+                    Format Excel
+                </a>
+                <a class="button" href="{{ request.resource_url(context.parent, 'download_amendements', query={'format': 'pdf'}) }}">
+                    <i class="fa fa-file-pdf"></i>
+                    Format PDF
+                </a>
+            </div>
+        </div>
+        <div>
+            <h3>Supprimer</h3>
+            <p>Supprimer la lecture ainsi que tous les amendements et réponses associés.</p>
+            <form method="post" action="{{ request.resource_url(context.parent) }}">
+                <input type="submit" name="delete" class="danger button" value="Supprimer">
+            </form>
+        </div>
+    </section>
 
-    <table class="table table-striped">
-        <thead>
-            <tr>
-                <th></th>
-                <th>
-                    Article
-                    <input id="article-filter" class="form-control form-control-sm" type="text" placeholder="Nº">
-                </th>
-                <th>Nº</th>
-                <th class="nosort">Objet</th>
+    <table class="table" style="width:100%">
+        <thead class="box">
+            <tr class="headers">
+                <th class="nosort"></th>
+                <th width="15%">Article</th>
+                <th width="15%">Nº</th>
                 <th>Auteur</th>
                 <th>Groupe</th>
-                <th>Sort</th>
                 <th>
```

Comment:
Maybe we should force the `width` here too to avoid the moving parts when we open the advanced filters?

Reply:
Sounds good.